### PR TITLE
Added completed compulsory tasks

### PR DESCRIPTION
**Pull request by irynazalata**
Additional task is not completed yet

### CODE REVIEW

File: users/users.controllers.js [hunk 1]
```diff
@@ -18,15 +24,49 @@ async function createUser(req, res) {
...
+      plugins: [
+        imageminPngquant({
+          quality: [0.6, 0.8],
+        }),
+      ],
+    });
+
     const user = await User.create({
       ...body,
+      avatarURL: `http://localhost:3000/images/${avatarName}.png`,
```
Comment (NadyaHristuk) [inline users/users.controllers.js:58]: const PORT = process.env.port || 8080;
    `http://localhost:${PORT}/images/${avatarName}`;

File: users/users.controllers.js [hunk 2]
```diff
@@ -163,6 +203,27 @@ async function updateUserSubscription(req, res) {
...
   }
 }
+
+async function updateAvatar(req, res) {
+  try {
+    const { _id } = req.user;
+    const { filename } = req.file;
+    const updatedUser = await User.findByIdAndUpdate(
+      _id,
+      { avatarURL: `http://localhost:3000/images/${filename}` },
```
Comment (NadyaHristuk) [inline users/users.controllers.js:213]: http://localhost:${PORT}/images/${avatarName};

File: index.js [hunk 3]
```diff
@@ -31,11 +46,15 @@ function connectMiddlewares(app) {
   app.use(express.json());
   app.use(cors({ origin: '*' }));
   app.use(logger('dev'));
+  app.use(express.static('public'));
```
Comment (NadyaHristuk) [inline index.js:49]: app.use('/images', express.static('public/images'));

**Comment by irynazalata** [comment]
Done

**Pull request merged**